builtin/mainmenu/tab_about.lua

```diff
@@ -124,6 +128,12 @@ return {
 
 		local credit_list = {}
 		table.insert_all(credit_list, {
+			core.colorize("#222", "Dedication of the current release"),
```

Comment:
The color is a little hard to read.

Reply:
Maybe an orange color (for better emphasis and less brightness than yellow):
```suggestion
			core.colorize("#fc0", "Dedication of the current release"),
```

Reply:
Agreed on the readability but the thought was to have a less happy color.